### PR TITLE
Fix cross-build

### PR DESCRIPTION
**Pull request by nee-san**
Fixes in make file to support cross-build because maketab is a host tool and should be compiled with host compiler

### CODE REVIEW

**Comment by arnoldrobbins** [comment]
Hi. Just out of curiosity, What is the cross-build environment you're using?  Are there any kind of standards for $(HOSTCC) vs. $(CC) vs. anything else?  Thanks.

**Comment by nee-san** [comment]
I use my own cross compiler. HOSTCC used to compile host tools that needed to build target program. If you build host tool with cross-compiler, onetrueawk's build system will run the binary and give error like "No such file or directory" or "Binary format error".

**Comment by nee-san** [comment]
CC used to compile target binary. It does not have to be a cross compiler

File: makefile [hunk 1]
```diff
@@ -31,6 +31,7 @@ CFLAGS = -O2
 #CC = gcc -O4 -Wall -pedantic -fno-strict-aliasing
 #CC = gcc -fprofile-arcs -ftest-coverage # then gcov f1.c; cat f1.c.gcov
 CC = gcc -g -Wall -pedantic 
+HOSTCC = gcc -g -Wall -pedantic
```
Comment (arnoldrobbins) [inline makefile]: I might suggest:
    `HOSTCC = gcc -g -Wall -pedantic`
    `CC = $(HOSTCC)     # change this is cross-compiling.`

Comment (nee-san) [reply makefile]: done!

**Pull request merged**